invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/details/browse/index.html

```diff
@@ -0,0 +1,34 @@
+{# -*- coding: utf-8 -*-
+
+This file is part of Invenio.
+Copyright (C) 2024 CERN.
+
+Invenio is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_communities/details/base.html" %}
+
+{% set active_community_header_menu_item= 'browse' %}
+{% set api_endpoint = community["links"]["subcommunities"] %}
```

Comment:
```suggestion
{% set subcommunities_endpoint = community["links"]["subcommunities"] %}
```

In the browse page we'll need to render multiple things, so I'd make it very explicit what the endpoint is.